libgadget/run.c

```diff
@@ -158,7 +158,7 @@ run(int RestartSnapNum)
     gravpm_init_periodic(&pm, All.BoxSize, All.Asmth, All.Nmesh, All.G);
 
     DomainDecomp ddecomp[1] = {0};
-    init(RestartSnapNum, ddecomp);          /* ... read in initial model */
+    inttime_t Ti_Current = init(RestartSnapNum, ddecomp);          /* ... read in initial model */
```

Comment:
Suggest using an inttime_t * on the args as return value, just in case we may return more down the road.

Reply:
I think we may end up doing that, but let me defer it a little. The time variables are a bit messed up and complicated: I would like to move them out of allvars.h and this may finish with them outside of init altogether.